Example_DynamicFramework/Demo/AppDelegate.m

```diff
@@ -18,7 +18,7 @@
 #import "AppDelegate.h"
 #import "SampleCaseViewController.h"
 #import "AppKeyChoiceViewController.h"
-
+#import <Analytics/SEGAnalytics.h>
```

Comment:
In this case, streethawk must depends on segment to work. You need to add this into pod's dependency. Now it depends on customer to add "pod 'Analytics', '~> 3.0'".

Reply:
More importantly, this file need to be added into Pointzi built output library. Check how AFNetworking does.

Reply:
I think carefully, it's too heavy to add additional dependency only for `[[SEGAnalytics sharedAnalytics] getAnonymousId]`. We can use `performSelector` to archive this.

Reply:
I think @Xingyuj commited his experimental code ? this is not required

Reply:
Alright, I thought I should add the segment for our demo app as well, I will get rid of those changes